boost-common/src/test/java/io/openliberty/boost/common/utils/BoosterUtil.java

```diff
@@ -5,18 +5,48 @@
 
 import io.openliberty.boost.common.BoostException;
 import io.openliberty.boost.common.boosters.AbstractBoosterConfig;
+import io.openliberty.boost.common.boosters.CDIBoosterConfig;
 import io.openliberty.boost.common.boosters.JDBCBoosterConfig;
+import io.openliberty.boost.common.boosters.JSONPBoosterConfig;
+import io.openliberty.boost.common.boosters.MPConfigBoosterConfig;
+import io.openliberty.boost.common.boosters.MPHealthBoosterConfig;
+import io.openliberty.boost.common.boosters.MPOpenTracingBoosterConfig;
+import io.openliberty.boost.common.boosters.MPRestClientBoosterConfig;
 
 public class BoosterUtil {
-    
-    private static Map<String, String> createDependenciesWithBoosterAndVersion(Class<?> booster, String version) throws BoostException {
+
```

Comment:
If BoosterUtil is only used in test, should we move it into src/test/java to make it clearer that these are not APIs used by the plugin during a build?

It seems like the naming is off too.. if we're going to have 0.1 and 0.2 for EE 7 and EE 8 for, say, CDI... then `getCDIDependency` is the wrong name...   Maybe just get rid of this name and call `createDependenciesWithBoosterAndVersion` with the arguments each time ?   Or I guess if JDBC is a special case you could leave it but don't follow this pattern.

Reply:
Why did I think this wasn't already in 'test'?  No idea..sorry

Reply:
i don't think there is anything special about jdbc, but just to be sure i am going to leave that one as is and make the suggested changes for the ones i introduced via this pr.